packages/terra-form-textarea/src/terra-dev-site/doc/example/DisabledWithError.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import Textarea from 'terra-form-textarea/lib/Textarea';
+
+const DisabledWithErrorExample = () => (
```

Comment:
Would this be a valid example we would want to show? I would think a disabled textarea shouldn't technically be able to reach the invalid state

Reply:
Thoughts @neilpfeiffer?

Reply:
From a usability perspective, no, an input or text area "should" never be disabled and invalid since you would never be able to resolve unless it was only temporary while focus was on another item/field/input then returned to active once the other item was resolved. 

But from a basic UI perspective, since it is something that could exist in technical implementation by combining props/styles, we should both be sure to handle it, and we should have a test and screenshot for that case, but I don't think we need it as a main example on the docsite page.

Reply:
@emilyrohrbough  Do you want me to go ahead and remove this example from the site or leave it? Let me know what other changes you'd like here.

Reply:
@sharynneazhar Lets remove this from the documentation pages. Since this isn't a desirable use-case we don't want to confuse consumers by providing an example we don't want people to implement.

Reply:
729b016